tests/cmdline/commands/test_storage.py

```diff
@@ -133,27 +132,21 @@ def mock_maintain(*args, **kwargs):
         for key, val in kwargs.items():
             log_message += f' > {key}: {val}\n'
 
-        logging.info(log_message)
+        AIIDA_LOGGER.report(log_message)
 
     monkeypatch.setattr(storage, 'maintain', mock_maintain)
 
-    with caplog.at_level(logging.INFO):
```

Comment:
It was INFO level, but in new code it seems the default level `REPORT` is used?

Reply:
Yeah, the INFO level was not necessary, because all the messages that are being tested for are anyway logged on the REPORT level, so we can use the default. That's why the test also simply pass even after removing the caplog.at_level line